docker/test

```diff
@@ -54,6 +66,15 @@ while getopts ":c:mn:pxqlde" opt; do
   esac
 done
 
+COMPOSE_ARGS="$COMPOSE_ARGS-$OPERATION_SYSTEM.yml"
+
+if ! [ $COMPOSE_ARGS ];
+then
+  echo "Docker compose file does not exist"
```

Comment:
Since this error is the result of someone choosing an operating system that isn't supported, like `-o cs10`, can we change this to something like `Operating system not supported`?

Reply:
Done, I also print that bad os what user enter.